cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/AnalyzerUtils.java

```diff
@@ -153,6 +153,7 @@ public static int compare(double d1, double d2, double epsilon) {
    */
   public static Map<String, Goal> getCaseInsensitiveGoalsByName(KafkaCruiseControlConfig config) {
     List<Goal> goals = config.getConfiguredInstances(KafkaCruiseControlConfig.GOALS_CONFIG, Goal.class);
+    goals.addAll(config.getConfiguredInstances(KafkaCruiseControlConfig.INTRA_BROKER_GOALS_CONFIG, Goal.class));
```

Comment:
If it is not relevant here (Note that we don't seem to have a sanity check to ensure that this config is not empty), should we refrain from adding these goals in this branch?  -- we may limit this line to `migrate_to_XXX` branch.